core/src/main/java/org/apache/accumulo/core/client/AccumuloClient.java

```diff
@@ -338,6 +338,13 @@ ConditionalWriter createConditionalWriter(String tableName, ConditionalWriterCon
   @Override
   void close();
 
+  /**
+   * Sets a user-defined ClientThreadPools implementation
+   *
+   * @param impl
```

Comment:
javadoc quality: remove tag if no description, or add description

Reply:
Fixed in 94a6163, I should have waited to have you review until the checks were done.